akka-stream/src/main/scala/akka/stream/impl/fusing/FutureFlow.scala

```diff
@@ -31,7 +32,7 @@ import scala.util.{ Failure, Success, Try }
       //seems like we must set handlers BEFORE preStart
       setHandlers(in, out, Initializing)
 
-      override def preStart(): Unit = {
+      override def preStart(): Unit =
```

Comment:
I actually prefer the curly brackets for procedure/imperative methods like this vs single-expression-methods where I think not needing them is very nice. Not very important though.